tests/apps/lwaftr/data/add-dot1q.py

```diff
@@ -0,0 +1,27 @@
+#! /usr/bin/env python2
+
+import sys
+from scapy.all import rdpcap, wrpcap, NoPayload, Ether, Dot1Q
+
+if len(sys.argv) != 3:
+    raise SystemExit("Usage: " + sys.argv[0] + " input.pcap output.pcap")
+
+packets = []
+for packet in rdpcap(sys.argv[1]):
+    layer = packet.firstlayer()
+    while not isinstance(layer, NoPayload):
+        if 'chksum' in layer.default_fields:
+            del layer.chksum
+        if (type(layer) is Ether):
```

Comment:
Nit: I don't know if parenthesis are strictly necessary here, there are not parens in the if above.